.vscode/launch.json

```diff
@@ -14,6 +14,14 @@
             "args": [],
             "env": {
                 "CONFIG_PATH": "${workspaceFolder}/configs",
+                "ENV_NAME": "local",
```

Comment:
The variable name `ENV_NAME` is rather generic. What are other possible values? Is it just for RBAC testing or is it used for other settings, too?

Reply:
I was wrong! I have updated the PR with this change! thanks!